### ISSUE TITLE
Recent update broke xterm

### ISSUE THREAD

**Issue by Root-voidX** [corbinlc/GNURootDebian#152]
After the recent update,

Gnuroot debian xterm won't launch on the s7 edge

**Comment by Root-voidX** [comment]
Also exiting the terminal causes force close

**Comment by Root-voidX** [comment]
Reinstall didn't help either

**Comment by corbinlc** [comment]
Can you try the new beta version 0.6.10?

**Comment by corbinlc** [comment]
You are obviously already getting beta versions, so you just need to update

**Comment by Root-voidX** [comment]
The new beta version is installed. Xterm launched, also launched octave via xterm using octave --force-gui so gui apps seem to work.

Though when exiting gnuroot debian it closes but Android tells me that its not responding/force close. If I click okay it will bring the same message up and then finally its gone

**Comment by Root-voidX** [comment]
Yes the force close happens only when exiting xterm. Using just the command line is fine. Does anyone else have this issue?

**Comment by corbinlc** [comment]
What exactly are you closing?  Just the xterm or the graphical environment (bvnc client) or the regular term that the vncserver was started from?  How are you closing what you are closing?  I am not seeing an issue, but I am not sure I am doing things like what you are doing.

**Comment by Root-voidX** [comment]
Okay so what I do is when im in xterm I use the 3 dotted menu to exit the xterm. Then I close the terminal that the vnc started from. It closes fine at first but after I get multiple force close popup from Android

**Comment by corbinlc** [comment]
So you hit "disconnect" from the menu within the vnc client? Does that by itself cause the "force close" dialog to appear, or does it only appear after you close the term that started the vnc server?

**Comment by Root-voidX** [comment]
The "unfortunately gnuroot debian has stopped dialog" only occurs after I close the vnc server term and other other terminal running.

**Comment by Root-voidX** [comment]
Also there is a warning displayed on the terminal while closing but its to fast I can't seem to screen shot it

**Comment by corbinlc** [comment]
That is weird. I don't think we touched anything that would effect exiting of the terminal emulator.  Did you used to follow the same flow for exiting the graphical environment and the terminal emulator with previous versions?  Did that always work for you without a dialog appearing?

**Comment by Root-voidX** [comment]
Yes version 0.6.8 I had no such issues, there was still this warning displayed on the terminal while closing but it was to quick to read, thought it was normal

**Comment by corbinlc** [comment]
If you just open a couple of terms, but don't start the graphical environment, and then exit those terms do you get the same "force close" dialog?

**Comment by corbinlc** [comment]
The little bit of text printed out is normal, but the "force close" is not normal.

**Comment by Root-voidX** [comment]
Okay so when closing multiple terms that bit of output before closing is normal yes? 

So if I close multiple normal terminals its fine, I don't get the not responding dialog. Add in a xterm I do get it

**Comment by corbinlc** [comment]
I have a thought that a new service that was added needs to be killed off when exiting.  It was added by someone else, so I have to look into the details, but that is probably what is going on.  We will try to fix this in the next beta release.

**Comment by corbinlc** [comment]
Can you try this APK?  https://www.dropbox.com/s/qyy8i7lmi4tylmr/GNURootDebianSource-armhf-release.apk?dl=0
You may need to uninstall the old version if it won't let you replace the existing one because their version numbers are the same, but I think it should work fine.

**Comment by Root-voidX** [comment]
I'll sideload it when I get home. My data connection at the moment is hit or miss

**Comment by Root-voidX** [comment]
Just installed it! Looks good. 

I opened xterm, used a gui app and closed. No force close.

I opened multiple terms with different ApplicTions and a. Xterm and closed them. No force close, though saw that warning message before closing. Idk what you did but its working. Anything else you want me to test ?

**Comment by corbinlc** [comment]
The newest beta version, 0.6.11 has the new features with the fix in it.  So, it should be effectively what I gave you.  Can you try it out and confirm that it still works for you?  Thanks for all your feedback.

**Comment by Root-voidX** [comment]
Looks good. No force closes. I've been testing chroot as well as proot. I noticed in chroot that scilab runs fine but not in proot. Is proof missing java ? Thought I would question it here instead of new issue !

**Comment by ondoisme** [comment]
Hi sir, please help me how to stop vnc client from auto running when gnu root debian started, sorry for my bad english.

Thanks so much

**Comment by AdamMatthewLTM** [comment]
Hi all,

I am currently assisting the developers in sorting relevant issues for GNURoot Debian on GitHub. If the issue discussed is still relevant, could you please reply to this thread and state the issue needs to be resolved and I will attempt to sort it for you ASAP. If the issue is no longer relevant, please reply to this thread and state it has been resolved, and I will suggest this thread be closed.

Furthermore, if a particular issue is considered to be high priority, please state that you are requesting it to be considered as high priority and I will mark it as such.

Thank you all so much for your time and understanding,
-Adam
